jbpm-recommendation-pmml-random-forest/pom.xml

```diff
@@ -50,6 +50,12 @@
           <groupId>org.jbpm</groupId>
           <artifactId>jbpm-test</artifactId>
           <version>${version.org.kie}</version>
```

Comment:
it's not part of the PR itself, but since I notice `${version.org.kie}` it would be nice to remove it
```suggestion
```

Reply:
Hi @Ginxo, thanks for dedicating some time to review this PR!

There are lots of `<version>${version.org.kie}</version>` in the `pom.xml` files of this repository.
I tried to simply remove them but maven complains about missing versions.
Probably other changes are necessary to achieve this goal.
Can you please create a JIRA ticket and assign it accordingly?